src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -13,4 +13,35 @@
     <Compile Include="System.Runtime.CompilerServices.Unsafe.il" />
     <Reference Include="System.Runtime" />
   </ItemGroup>
+
+  <Target Name="AddResourcesFileToIlasm" BeforeTargets="CoreCompile" Condition="'$(OS)'=='Windows_NT'">
+    <MakeDir Directories="$(IntermediateOutputPath)Disassembly" />
+
+    <PropertyGroup>
+      <_IldasmCommand>$(ToolsDir)ilasm/ildasm.exe</_IldasmCommand>
+      <_IldasmCommand>$(_IldasmCommand) $(ContractOutputPath)/$(MSBuildProjectName).dll</_IldasmCommand>
+      <_IldasmCommand>$(_IldasmCommand) /OUT=$(IntermediateOutputPath)Disassembly/$(MSBuildProjectName).il</_IldasmCommand>
+
+      <_CvtResCommand>cvtres.exe</_CvtResCommand>
```

Comment:
You can consider checking for this in the framework directory (`$(SystemRoot)\Microsoft.NET\Framework\v4.0.30319\cvtres.exe`) and falling back to assume on the path and warning if it is absent.  That way you can support a case where someone ran outside a developer cmd prompt (even though its technically a requirement).